targets/lenovo-x1/guivmExtraModules.nix

```diff
@@ -150,10 +153,15 @@
       };
     };
 
-    # Open TCP port for the PDF XDG socket
-    networking.firewall.allowedTCPPorts = [xdgPdfPort];
+    # Enable all firmware for graphics firmware
+    hardware.enableRedistributableFirmware = true;
+    hardware.enableAllFirmware = true;
     # Early KMS needed for GNOME to work inside GuiVM
     boot.initrd.kernelModules = ["i915"];
+    boot.kernelParams = ["i915.force_probe=a7a1" "earlykms"];
```

Comment:
`"i915.force_probe=a7a1"` - this is now hardcoded for **x1 gen11** only, probably best to get the current passthrough gpu device. Also, does this achieve anything?